src/dom/dom.js

```diff
@@ -2129,8 +2140,8 @@ p5.Element.prototype.hide = function() {
  */
 /**
  * @method size
- * @param  {Number|Constant} w    width of the element, either AUTO, or a number
- * @param  {Number|Constant} [h] height of the element, either AUTO, or a number
+ * @param  {Number|Constant} w    width of the element, either a number, or `AUTO`
+ * @param  {Number|Constant} [h] height of the element, either a number, or `AUTO`
```

Comment:
This line and the line above cannot be changed because its structure is used to infer types in the documentation generation process. See more [here](https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md#specify-parameters).